Src/FluentAssertions/Primitives/DateTimeAssertions.cs

```diff
@@ -715,6 +715,62 @@ public AndConstraint<TAssertions> NotHaveSecond(int unexpected,
         return new AndConstraint<TAssertions>((TAssertions)this);
     }
 
+    /// <summary>
+    /// Asserts that the current <see cref="DateTime"/>  has the <paramref name="expected"/> millisecond.
+    /// </summary>
+    /// <param name="expected">The expected milliseconds of the current value.</param>
+    /// <param name="because">
+    /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+    /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+    /// </param>
+    /// <param name="becauseArgs">
+    /// Zero or more objects to format using the placeholders in <paramref name="because" />.
+    /// </param>
+    public AndConstraint<TAssertions> HaveMillisecond(int expected,
```

Comment:
🤔 I feel that `HaveMilliseconds` is more proper English.

Reply:
I thought it would be more consistent with the existing APIs, for example, `HaveMinute` and `HaveSecond`.

Reply:
Yeah, I figured that, but it didn't feel correct. After "consulting" with ChatGPT I'm more convinced that `HaveMilliseconds` is the right name.